junit4/src/main/java/com/linecorp/armeria/testing/junit4/server/ServerRule.java

```diff
@@ -273,4 +288,19 @@ public InetSocketAddress httpSocketAddress() {
     public InetSocketAddress httpsSocketAddress() {
         return delegate.httpsSocketAddress();
     }
+
+    /**
+     * Returns the {@link WebClient}.
+     */
+    public WebClient webClient() {
+        return delegate.webClient();
+    }
+
+    /**
+     * Creates a {@link WebClient} each time with {@link WebClientBuilder}.
+     */
+    public WebClient webClient(Consumer<WebClientBuilder> webClientCustomizer) {
+        requireNonNull(webClientCustomizer, "webClientCustomizer");
+        return delegate.webClient(webClientCustomizer);
```

Comment:
Question: It seems like `configureWebClient` is eventually called in `delegate.webClient(...)`. 
Is it okay or should we just apply `webClientCustomizer`?

Reply:
I was thinking that configureWebClient is a common configuration, webClientCustomizer is an individual configuration. Otherwise, I don't think it looks useful like the previous code.

WDYT? 🤔

Reply:
Maybe I don't fully understand what you meant. 🤣 
So are you saying it should be okay to apply both configurations?

Reply:
Yes, I thought so 🤣

Reply:
Sounds good to me as well. 😄